packaging/ubuntu-14.04/snapd.postrm

```diff
@@ -108,6 +108,9 @@ if [ "$1" = "purge" ]; then
         umount -l /run/snapd/ns/ || true
     fi
 
+    echo "Unmounting snapd-specific naming cgroup"
+    umount -l /run/snapd/cgroup || true
```

Comment:
Ditto. BTW, why are we maintaining 14.04 if snapd no longer supports it?

Reply:
I don't know anything about that. If we no longer intend to ship snapd to 14.04 then we need store side mechanics to prevent snapd on 14.04 from refreshing. In either case I think this is separate from this pull request.

Reply:
@mvo5 confirmed that 14.04 is still supported.